pim-tool-back/src/main/java/vn/elca/training/model/entity/ProjectEmployee.java

```diff
@@ -0,0 +1,18 @@
+package vn.elca.training.model.entity;
+
+import javax.persistence.*;
+
+@Entity
+public class ProjectEmployee {
```

Comment:
[N] If there is no other columns than Ids in the join table. Just use @ManyToMany, it makes managing the relationship much easier.